### PR TITLE
fix merge dict bools

### PR DESCRIPTION
**Pull request by JarbasAl**
the check for skip_empty flags would ignore False booleans

### CODE REVIEW

File: ovos_utils/json_helper.py [hunk 1]
```diff
@@ -26,7 +26,7 @@ def merge_dict(base, delta, merge_lists=False, skip_empty=False,
         else:
             if new_only and k in base:
                 continue
-            if skip_empty and not d:
+            if skip_empty and not d and d is not False:
```
Comment (ChanceNCounter) [inline ovos_utils/json_helper.py]: Will this also skip `0`? Maybe it should just be `d is None`

Comment (JarbasAl) [reply ovos_utils/json_helper.py]: i think it should also skip empty lists and empty strings, but you are right that 0 should not be skipped

perhaps a check for isinstance ?

Comment (ChanceNCounter) [reply ovos_utils/json_helper.py]: `all((i == None for i in {}))` will return correctly on iterables.

**Comment by ChanceNCounter** [comment]
That's weird, my notifications menu showed this purple already. I was assuming I pushed the button and forgot about it :/

**Pull request merged**